### PR TITLE
Update docs for development environment

### PR DESCRIPTION
**Pull request by thommay**
:construction: I'm updating the docs as I go, but here's the first bit :construction:

### CODE REVIEW

File: dev/README.md [hunk 1]
```diff
@@ -50,6 +50,31 @@ In a separate terminal session/pane/window:
...
+To use your running Chef Server through standard commands such as knife,
+you'll need to create an organisation and a user, and then create a
+knife config on your workstation.
+
+    vagrant ssh
+    sudo -i
+    # create a user to access chef with
+    chef-server-ctl user-create -f /tmp/admin.pem admin Admin User admin@example.com password 
+    # create an organization
+    chef-server-ctl org-create -f /tmp/test-validator.pem test Test
```
Comment (stevendanna) [inline dev/README.md]: You can add `-a admin` and the user will automatically be added to the org so you don't need the step below.

**Comment by stevendanna** [comment]
Left a minor comment, but this looks good to me.  Thanks for taking the time to update these.

File: dev/README.md [hunk 2]
```diff
@@ -50,6 +50,31 @@ In a separate terminal session/pane/window:
...
+    # associate the user with the organization
+    chef-server-ctl org-user-add test admin
+
+Now on your workstation, create `.chef/knife.rb` in the root of your
```
Comment (marcparadise) [inline dev/README.md]: Maybe we could pre-create this in dev/.chef as part of this PR?

**Comment by stevendanna** [comment]
> Maybe we could pre-create this in dev/.chef as part of this PR?

I think let's merge this.  It is better than no documentation and we can do the precreation when we have time.

**Pull request merged**